ts_src/psbt.ts

```diff
@@ -769,16 +769,19 @@ export class Psbt {
         prevout = { ...input.witnessUtxo! };
       }
 
-      const unblindPrevout = getBlindingDataForInput(
-        prevout,
-        blindingPrivkeys[index],
-      );
-
-      if (unblindPrevout) {
+      const privKey = blindingPrivkeys[index]
+      if (privKey) {
+        const unblindPrevout = getBlindingDataForInput(prevout, privKey);
         inputAgs.push(unblindPrevout.ag);
         inputValues.push(unblindPrevout.value);
         inputAbfs.push(unblindPrevout.abf);
         inputVbfs.push(unblindPrevout.vbf);
+      } else {
+        const zeroBuffer32bytes = Buffer.alloc(32)
+        inputValues.push(confidential.confidentialValueToSatoshi(prevout.value).toString(10))
+        inputAgs.push(prevout.asset.slice(1))
+        inputAbfs.push(zeroBuffer32bytes)
+        inputVbfs.push(zeroBuffer32bytes)
```

Comment:
why not returning this into `getBlindingDataForInpit` in case the private key is undefined? It would increase readability. Also, you could be able to use the ZERO constant of transaction module instead of instantiating this zeroBuffer32Bytes.